### PR TITLE
Added Slovenian translations

### PR DESCRIPTION
**Pull request by valjar**

### CODE REVIEW

File: src/Formatter.php [hunk 1]
```diff
@@ -32,6 +34,14 @@ class Formatter implements IFormatter
 			'money' => '%currency %money',
 			'date' => 'd/m/Y',
 		],
+        'si' => [
```
Comment (MartkCz) [inline src/Formatter.php:37]: Please, use tabs instead of spaces

File: src/Translator.php [hunk 2]
```diff
@@ -10,7 +10,8 @@ class Translator implements ITranslator
 	use SmartObject;
 
 	public const ENGLISH = 'en',
-		CZECH = 'cs';
+		CZECH = 'cs',
+        SLOVENIAN = 'si';
```
Comment (MartkCz) [inline src/Translator.php:15]: Please, use tabs instead of spaces

File: src/Translator.php [hunk 3]
```diff
@@ -66,6 +67,32 @@ class Translator implements ITranslator
 			'subtotal' => 'Subtotal',
 			'dueDate' => 'Due date',
 		],
+        'si' => [
```
Comment (MartkCz) [inline src/Translator.php:71]: Please, use tabs instead of spaces

**Review by adaamz — CHANGES_REQUESTED** [review]
use tabs everywhere, not only on those 3 mentioned places

**Pull request closed without merging**